src/Metrics.php

```diff
@@ -72,9 +72,15 @@ public function declare(string $name, CollectorInterface $collector): void
         }
     }
 
-    public function unregister(string $name): void
+    public function unregister(string $name, string $namespace = ''): void
     {
+        \assert($name !== '');
```

Comment:
`MetricsInterface::unregister` throws only `MetricsException`. Your change throws `AssertionError`, which not implements `MetricsException`.
`MetricsException` describe RPC errors. I think need create new exception class, maybe `UnregisterException` or just throws `LogicException` and add `throws \LogicException` to interface